acceptance/tests/environment/custom_type_provider_from_same_environment.rb

```diff
@@ -2,6 +2,10 @@
 require 'puppet/acceptance/environment_utils'
 extend Puppet::Acceptance::EnvironmentUtils
 
+tag 'audit:medium',
+    'audit:acceptance',
```

Comment:
This seems like it could be an integration test, and if it says might be able to use `mk_temp_environment_with_teardown`?